lib/node_modules/@stdlib/utils/is-lowercase/test/test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isLowercase = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof isLowercase, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `true` if provided a lowercase string', function test( t ) {
+	var values;
+	var bool;
+	var i;
+
+	values = [
+		'hello world!',
+		'above all summits it is calm'
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		bool = isLowercase( values[ i ] );
+		t.equal( bool, true, 'returns true when provided '+values[ i ] );
+	}
+	t.end();
+});
+
+tape( 'the function returns `false` if not provided a lowercase string', function test( t ) {
+	var values;
+	var i;
+
+	values = [
+		'In all the tree-tops you feel scarcely a breath',
+		'HELLO WORLD',
+		void 0,
+		0,
+		NaN,
+		null,
+		false,
+		true,
+		[],
+		{},
+		function(){}
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.equal( isLowercase( values[i] ), false, 'returns false when provided '+values[i] );
+	}
+	t.end();
+});
```

Comment:
If an empty `string` is supposed to pass, should also include a test stating such behavior.